.travis/before_script_test.sh

```diff
@@ -1,3 +1,9 @@
 #!/usr/bin/env bash
 set -ev
 
+TESTING_SCRIPTS_DIR=vendor/symfony-cmf/testing/bin
+CONSOLE=${TESTING_SCRIPTS_DIR}/console
+
+@echo '+++ create PHPCR +++'
+${CONSOLE} doctrine:phpcr:init:dbal --drop --force
+${CONSOLE} doctrine:phpcr:repository:init
```

Comment:
Please configure your IDE to always add a newline at end of file, this will make the red pictogram disappear.